### PR TITLE
new notepad script

### PR DESCRIPTION
**Pull request by joelwass**

### CODE REVIEW

**Comment by stripedpajamas** [comment]
can you add route for np as well?

**Comment by joelwass** [comment]
oh baby

**Comment by joelwass** [comment]
this might get spicayyyy

**Comment by joelwass** [comment]
still working on this hold on

File: index.js [hunk 1]
```diff
@@ -5,12 +5,15 @@ const er = require('./payloads/er');
 const al = require('./payloads/al');
 const ls = require('./payloads/ls');
 const lm = require('./payloads/lm');
+const np = require('./payloads/np');
+const friday = require('./payloads/friday');
```
Comment (stripedpajamas) [inline index.js]: Change this to `fr`

**Pull request merged**